share/man/man4/pfsync.4

```diff
@@ -147,7 +147,6 @@ E.g.:
 # ifconfig pfsync0 version 1301
 .Ed
 .Pp
-.Bd
```

Comment:
This tag was ignored because it didn't have a `-type`.  None of the display types really seemed necessary, so I opted for keeping the output unchanged (while fixing the syntax error).